### DISCUSSION TITLE
افزودن ایپی دلخواه

### DISCUSSION THREAD

**Discussion by sirvan1133** [MortezaBashsiz/CFScanner#354] [Ideas]
اگه بشه یه لیست اضافه کنیم برای تست در برنامه عالیه 
برای مثال ما تعدادی ایپی داریم که جمع اوری کردیم و در یک فایل تکست ذخیره کردیم
اگه بشه برای تست وارد برنامه کرد عالی میشه

**Comment by goingfine** [comment]
در نسخه ویندوزی از منوی فایل گزینه import رو بزنید

**Comment by tempookian** [comment]
روی نسخه‌های پایتون و گو هم این امکان وجود داره
